README.md

```diff
@@ -31,7 +31,7 @@ Code Smells, or maintainability issues, are raised for places of code which migh
 * Boolean literals should not be redundant ([`no-redundant-boolean`])
 * "switch" statements should have at least 3 "case" clauses ([`no-small-switch`])
 * "catch" clauses should do more than rethrow ([`no-useless-catch`])
-* Local variables should not be declared and then immediately returned or thrown ([`prefer-immediate-return`]) (:wrench: *fixable*)
+* Local variables should not be declared and then be immediately returned or thrown ([`prefer-immediate-return`]) (:wrench: *fixable*)
```

Comment:
I disagree with this change

Reply:
Yeah, not sure what I was thinking here...